### ISSUE TITLE
Map new 1.18 biomes

### ISSUE THREAD

**Issue by SlimeDog** [SlimeDog/BiomeRemap#5]
**Description**
Add to BiomeRemap boilerroom
<table>
<tr><td> grove                           </td><td> replacement-biome: meadow       </td></tr>
<tr><td> lofty_peaks                 </td><td> replacement-biome: mountains    </td></tr>
<tr><td> meadow                      </td><td> replacement-biome: meadow        </td></tr>
<tr><td> snow_capped_peaks </td><td> replacement-biome: mountains    </td></tr>
<tr><td> snowy_slopes             </td><td> replacement-biome: mountains    </td></tr>
<tr><td> stony_peaks               </td><td> replacement-biome: stony_peaks </td></tr>
</table>
Add to BiomeRemap snowbound
<table>
<tr><td> grove                           </td><td> replacement-biome: grove                               </td></tr>
<tr><td> lofty_peaks                 </td><td> replacement-biome: lofty_peaks                     </td></tr>
<tr><td> meadow                       </td><td> replacement-biome: snowy_tundra                 </td></tr>
<tr><td> snow_capped_peaks  </td><td> replacement-biome: snow_capped_peaks     </td></tr>
<tr><td> snowy_slopes             </td><td>  replacement-biome: snowy_slopes                 </td></tr>
<tr><td> stony_peaks                </td><td> replacement-biome: snowy_slopes </td></tr>
</table>

**Comment by SlimeDog** [comment]
Change made in branch=one18, pending merge to release for MC 1.18.

**Comment by SlimeDog** [comment]
More changes were added in 1.18-21w40a.

### Renamed biomes
Fourteen biomes were renamed in Minecraft 1.18-21w40a.
```
      frozen_peaks                      formerly snowcapped_peaks
      jagged_peaks                      formerly lofty_peaks
      old_growth_birch_forest           formerly tall_birch_forest
      old_growth_pine_taiga             formerly giant_pine_taiga
      old_growth_spruce_taiga           formerly giant_spruce_taiga
      snowy_plains                      formerly snowy_tundra
      sparse_jungle                     formerly jungle_edge
      stony_shore                       formerly stone_shore
      windswept_forest                  formerly wooded_mountains
      windswept_gravelly_hills          formerly gravelly_hills
      windswept_hills                   formerly extreme_hills
      windswept_hills                   formerly mountains
      windswept_savanna                 formerly shattered_savanna
      wooded_badlands                   formerly wooded_badlands_plateau
```
### Deleted biomes
Twenty-five biomes were deleted from the code in Minecraft 1.18-21w40a.
```
      badlands_plateau
      bamboo_jungle_hills
      birch_forest_hills
      dark_forest_hills
      desert_hills
      desert_lakes
      giant_spruce_taiga_hills
      giant_tree_taiga_hills
      jungle_hills
      modified_badlands_plateau
      modified_gravelly_mountains 
      modified_jungle
      modified_jungle_edge
      modified_wooded_badlands_plateau 
      mountain_edge
      mushroom_field_shore
      shattered_savanna_plateau
      snowy_mountains 
      snowy_taiga_hills 
      snowy_taiga_mountains 
      swamp_hills
      taiga_hills
      taiga_mountain
      tall_birch_hills
      wooded_hills
```
Started with Minecraft 1.18, they cannot be used as replacement biomes in **BiomeRemap**.
The following changes will be made to the default replacement biomes:
```
      snowy_taiga_hills     changed to  snowy_taiga
      snowy_taiga_mountains changed to  snowy_slopes
```

**Comment by SlimeDog** [comment]
Updated config.yml in branch=one18, to conform with 1.18-21w40a. Further changes are anticipated.

**Comment by SlimeDog** [comment]
Updated config.yml in branch=one18, to conform with 1.18-21w43a (deep_warm_ocean deleted). Further changes are anticipated.

**Comment by SlimeDog** [comment]
Released BiomeRemap 3.0.0
